content/v2.0/security/https_setup.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Enabling HTTPS with InfluxDB
+description: >
+  Enable HTTPS and Transport Security Layer (TLS) secure communication between clients and your InfluxDB servers.
+weight: 201
+menu:
+  v2_0:
+    name: Enabling HTTPS
+    parent: Security & authorization
+v2.0/tags: [security, authentication, tls, https]
+---
+
+Enabling HTTPS encrypts the communication between clients and the InfluxDB server.
+When configured with a signed certificate, HTTPS can also verify the authenticity of the InfluxDB server to connecting clients.
+
+{{% warn %}}
+InfluxData [strongly recommends](/influxdb/v1.7/administration/security/) enabling HTTPS, especially if you plan on sending requests to InfluxDB over a network.
```

Comment:
So some bike-shedding here and I'm not sure exactly the best way to go. Users actually enable TLS which is what provides the secure encryption framework. Data sent over the HTTPS protocol utilizes that framework. So it's true that users "enable HTTPS," but it may be more accurate to say "enable TLS" and "use/force HTTPS protocol." Thoughts?

Reply:
I trend to stick with "Enable TLS over HTTPs"